src/devtools/views/Components/OwnersStack.js

```diff
@@ -1,43 +1,85 @@
 // @flow
-
-import React, { useCallback, useContext } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useState,
+  createRef,
+  forwardRef,
+} from 'react';
+import throttle from 'lodash.throttle';
+import classNames from 'classnames';
 import Button from '../Button';
 import ButtonIcon from '../ButtonIcon';
 import { TreeContext } from './TreeContext';
 import { StoreContext } from '../context';
+import { getElementDimensions } from '../../../utils';
 
 import type { Element } from './types';
 
 import styles from './OwnersStack.css';
 
-export default function OwnerStack() {
-  const { ownerStack, resetOwnerStack } = useContext(TreeContext);
+type ElementsDropdownProps = {
+  selectedElementIndex: number | null,
+  children: Array<any>,
+};
+function ElementsDropdown({
+  selectedElementIndex,
+  children,
+}: ElementsDropdownProps) {
+  const [isDropdownVisible, setIsDropdownVisible] = useState(false);
+  const handleClick = useCallback(() => {
+    setIsDropdownVisible(!isDropdownVisible);
+  }, [isDropdownVisible, setIsDropdownVisible]);
 
-  const elements = ownerStack.map((id, index) => (
-    <ElementView key={id} id={id} index={index} />
-  ));
+  useEffect(() => {
+    setIsDropdownVisible(false);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [selectedElementIndex]);
```

Comment:
I needed to disable this line unfortunately 😢. 
Main reason was that we need to track when we change selected elements. 
This way we can close the dropdown on every element change.